jobs/bodhi-trigger.Jenkinsfile

```diff
@@ -249,6 +249,9 @@ cosaPod(cpu: "0.1", kvm: false) {
                 --stream ${stream}
             """)
         }
+        if (test.result != 'SUCCESS') {
+            pipeutils.matrixSend("${currentBuild.description} - ${outcome}")
```

Comment:
I know you want to improve this in follow-ups, but at the very least to start we should add a link back to the job URL.